components/general/Blog/BlogGallery.module.css

```diff
@@ -0,0 +1,36 @@
+.blog_gallery {
+  padding: 63px 0 63px 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-bottom: var(--default-border);
+}
+
+.blog_gallery__large_group {
+  padding-bottom: 63px;
+  display: flex;
+  gap: 48px;
+  border-bottom: var(--default-border);
+}
+
+.blog_gallery__small_group {
+  padding-top: 63px;
+  display: flex;
+  flex-direction: column;
```

Comment:
How can you make it auto-fill to both ends without space-between?

<img width="1360" alt="image" src="https://user-images.githubusercontent.com/70834897/222004969-32f15b7a-e877-4f30-b203-b263cf4d0bf6.png">

Reply:
You look into wrong layer, I use the gap between two lines of small img group.